packages/base/src/icons/message-error.js

```diff
@@ -1,5 +1,5 @@
 import { registerIcon } from "../SVGIconRegistry.js";
-import { ICON_MESSAGE_ERROR } from "../../../main/src/i18n/defaults.js";
+import { ICON_MESSAGE_ERROR } from "../../../main/dist/generated/i18n/i18n-defaults";
```

Comment:
`.js`?

Reply:
shouldn't this be `@ui5/webcomponents/dist/generaated/i18n/...`?

Reply:
Added .js and leave it with relative path